apps/emqx_mq/test/emqx_mq_consumer_SUITE.erl

```diff
@@ -0,0 +1,57 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_mq_consumer_SUITE).
+
+-compile(nowarn_export_all).
+-compile(export_all).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("snabbkaffe/include/snabbkaffe.hrl").
+-include_lib("emqx/include/asserts.hrl").
+-include("../src/emqx_mq_internal.hrl").
+
+all() ->
+    emqx_common_test_helpers:all(?MODULE).
+
+init_per_suite(Config) ->
+    Apps = emqx_cth_suite:start(
+        [
+            {emqx_durable_storage, #{override_env => [{poll_batch_size, 1}]}},
```

Comment:
Note: this options will soon become obsolete. Is there a particular reason to set it?

Reply:
The reason is to have buffers both full and not full during the tests. With the default setting, the DS delivers large batches, we either have buffers filled at once of have to configure large buffers and prepare a lot of messages.